internal/plugins/helm/plugin.go

```diff
@@ -15,11 +15,17 @@
 package helm
 
 import (
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
 	"sigs.k8s.io/kubebuilder/pkg/plugin"
 )
 
 const pluginName = "helm" + plugin.DefaultNameQualifier
 
+var (
+	supportedProjectVersions = []string{config.Version2, config.Version3Alpha}
```

Comment:
We should make this plugin explicitly support only `3-alpha`+, otherwise we have no way to tell if this project is Helm or not because no config `layout` key exists (unless we hackily detect project type).
```suggestion
	supportedProjectVersions = []string{config.Version3Alpha}
```